src/SetsProfilePhotoFromUrl.php

```diff
@@ -16,7 +16,7 @@ trait SetsProfilePhotoFromUrl
     public function setProfilePhotoFromUrl(string $url)
     {
         $name = pathinfo($url)['basename'];
-        file_put_contents($file = sys_get_temp_dir().Str::uuid()->toString(), file_get_contents($url));
+        file_put_contents($file = sys_get_temp_dir().'/'.Str::uuid()->toString(), file_get_contents($url));
```

Comment:
YES! I'm getting this very issue now.
![image](https://user-images.githubusercontent.com/1200195/150261839-757a2096-efa1-4eb5-b655-d110a0646a10.png)